core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/deploysemantics/layer/WorkerExecution.scala

```diff
@@ -1,21 +1,22 @@
 package edu.uci.ics.amber.engine.architecture.deploysemantics.layer
 
-import edu.uci.ics.amber.engine.architecture.controller.WorkerPortExecution
+import edu.uci.ics.amber.engine.architecture.controller.execution.WorkerPortExecution
+import edu.uci.ics.amber.engine.architecture.worker.statistics.WorkerState.UNINITIALIZED
 import edu.uci.ics.amber.engine.architecture.worker.statistics.{WorkerState, WorkerStatistics}
-import edu.uci.ics.amber.engine.common.virtualidentity.ActorVirtualIdentity
 import edu.uci.ics.amber.engine.common.workflow.PortIdentity
 
 import scala.collection.mutable
 
 // TODO: remove redundant info
```

Comment:
this todo can be removed.

Reply:
removed. added another TODO for separation of state and stats.